src/index.ts

```diff
@@ -14,6 +16,14 @@ const PORT = process.env.PORT || 3000;
 app.use(express.json({ limit: '4mb' }));
 app.use(cors({ maxAge: 86400 }));
 app.use(compression());
+app.use(
+  morgan(
+    '[http] :remote-addr - :remote-user [:date[clf]] ' +
```

Comment:
We would need all these info? :)

Reply:
> Agree i have confusions with winston before, pain to use it in development, maybe there is some option to print logs in dev env too?

Error tracking system such as Sentry should do that. Will be on the next betting table

Reply:
HTTP logs are always useful. You know who send what, when, and the response. Current logs are only printing on error, we can not track the ratio of success vs error. You can also track spammer, and weird request who overload the server